src/coreclr/vm/eventing/eventpipe/ds-rt-coreclr.h

```diff
@@ -302,7 +302,7 @@ ds_rt_server_log_pause_message (void)
 {
 	STATIC_CONTRACT_NOTHROW;
 
-	const char diagPortsName[] = "DOTNET_DiagnosticPorts";
+	const char diagPortsName[] = "DiagnosticPorts";
```

Comment:
Thanks! However, now the print that happens below will fail to show the right name. Can you please add the `DOTNET_` prefix to the config on line 316?

Reply:
Is there a reason we can't use the standard `CLRConfig::EXTERNAL_DOTNET_DiagnosticPorts` instead of having an explicit string?

Reply:
> Is there a reason we can't use the standard `CLRConfig::EXTERNAL_DOTNET_DiagnosticPorts` instead of having an explicit string?

`CLRConfigNoCache::Get` seems like it only accepts ANSI but `CLRConfig::EXTERNAL_DOTNET_DiagnosticPorts` is Unicode.
This change was introduced on https://github.com/dotnet/runtime/commit/c3e0c8a3dcaa9be860953931fd1bf8cd579e8758

Reply:
I'd gotten to the same uft8 -> utf16 conclusion. Do you see a better way @davmason? Perhaps we could do something like `MAKE_UTF8PTR_FROMWIDE_NOTHROW` on the `ConfigStringInfo.name`? Don't know if it matters, but also doing that brings in all of utilcode. This is in the runtime specific part, so maybe that's not as bad.